bridge/device/cloud/manager.go

```diff
@@ -663,12 +373,15 @@ func (c *Manager) run() {
 }
 
 func (c *Manager) connect(ctx context.Context) error {
-	funcs := []func(ctx context.Context) error{
+	var funcs []func(ctx context.Context) error
+	if c.isCredsExpiring() {
+		funcs = append(funcs, c.refreshToken)
+	}
+	funcs = append(funcs, []func(ctx context.Context) error{
 		c.signUp,
-		c.refreshToken,
```

Comment:
I know why there was the refresh:
When device has been registered in hub and then the device has been restarted then the signup oparetion did nothing and refresh token updates the accesstoken (it can be invalid after the restart).

Did you cover this usecase with refresh token ?